iphone/Maps/UI/Search/Tabs/SearchTabViewController.swift

```diff
@@ -70,7 +70,10 @@ extension SearchTabViewController: SearchOnMapScrollViewDelegate {
 extension SearchTabViewController: SearchCategoriesViewControllerDelegate {
   func categoriesViewController(_ viewController: SearchCategoriesViewController,
                                 didSelect category: String) {
-    let query = SearchQuery(L(category) + " ", source: .category)
+    let preferredLang = AppInfo.shared().languageId
```

Comment:
This code correctly selects the first item from the union of a system's preferred languages and translations available in the app bundle.
```suggestion
    // Get the language of the translation displayed in the UI
    let appUILanguage = Bundle.main.preferredLocalizations.first
```